.github/workflows/do-release.yml

```diff
@@ -0,0 +1,158 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+name: Create draft release
+
+jobs:
+  create_release:
+    name: Create the Github Release
+    runs-on: ubuntu-latest
+    needs: [release-osx, release-ubuntu-16-04]
+    steps:
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          draft: true
+          prerelease: false
+      - uses: actions/download-artifact@v1
+        with:
+          name: sgrep-osx-${{ github.sha }}
+          path: sgrep-osx
+      - name: Get the version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+      - name: help
+        run: |
+          pwd;
+          ls;
+          find .;
+      - name: Upload Release Asset
+        id: upload-release-asset-osx
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: ./sgrep-osx/artifacts.zip
+          asset_name: sgrep-osx-${{ steps.get_version.outputs.VERSION }}.zip
+          asset_content_type: application/zip
+      - uses: actions/download-artifact@v1
+        with:
+          name: sgrep-ubuntu-16.04-${{ github.sha }}
+          path: sgrep-ubuntu
+      - name: Upload Release Asset
+        id: upload-release-asset-ubuntu
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: ./sgrep-ubuntu/artifacts.tar.gz
+          asset_name: sgrep-ubuntu-1604-${{ steps.get_version.outputs.VERSION }}.tar.gz
```

Comment:
the convention we've had with past releases has been `sgrep-SEMVER-ubuntu-16.04.tgz`